src/jit/lclvars.cpp

```diff
@@ -2689,6 +2689,10 @@ void Compiler::lvaSortByRefCount()
     lvaTrackedCount             = 0;
     lvaTrackedCountInSizeTUnits = 0;
 
+#ifdef DEBUG
+    VarSetOps::AssignNoCopy(this, lvaTrackedVars, VarSetOps::MakeEmpty(this));
```

Comment:
It would be a good idea to capture why this initialization is needed in debug only build here.